### ISSUE TITLE
Suspicious debug message from OCS Share API: cannot update etag on /files

### ISSUE THREAD

**Issue by PVince81** [owncloud/core#14370]
### Steps to reproduce
1. I ran the pyocclient test suite that includes creating/updating/deleting shares over the OCS Share PI

### Expected result
No suspicious messags

### Actual result:
```
{"reqId":"7h4jDWU0PGCiRTx5dgca","remoteAddr":"127.0.0.1","app":"files_sharing","message":"can not update etags on \/files\/ for user share. Path does not exists","level":0,"time":"2015-02-19T11:41:25+00:00","method":"POST","url":"\/owncloud\/ocs\/v1.php\/apps\/files_sharing\/api\/v1\/shares"}
{"reqId":"Jx1dk3n02j8vXZ3L9rJY","remoteAddr":"127.0.0.1","app":"files_sharing","message":"can not update etags on \/files\/ for user share. Path does not exists","level":0,"time":"2015-02-19T11:41:25+00:00","method":"DELETE","url":"\/owncloud\/ocs\/v1.php\/apps\/files_sharing\/api\/v1\/shares\/38"}
{"reqId":"u6pHhk0YtQDFVFS+vww1","remoteAddr":"127.0.0.1","app":"files_sharing","message":"can not update etags on \/files\/ for user share. Path does not exists","level":0,"time":"2015-02-19T11:41:26+00:00","method":"POST","url":"\/owncloud\/ocs\/v1.php\/apps\/files_sharing\/api\/v1\/shares"}
{"reqId":"IbSniHQ+cazcgBOadNnQ","remoteAddr":"127.0.0.1","app":"files_sharing","message":"can not update etags on \/files\/ for user share. Path does not exists","level":0,"time":"2015-02-19T11:41:27+00:00","method":"DELETE","url":"\/owncloud\/ocs\/v1.php\/apps\/files_sharing\/api\/v1\/shares\/39"}
{"reqId":"UrkRAGwPSFLp8zhVA2\/u","remoteAddr":"127.0.0.1","app":"files_sharing","message":"can not update etags on \/files\/ for user share. Path does not exists","level":0,"time":"2015-02-19T11:41:27+00:00","method":"POST","url":"\/owncloud\/ocs\/v1.php\/apps\/files_sharing\/api\/v1\/shares"}
```

### Versions
ownCloud 8.1 pre-alpha git master (84eb00e428693f504acd9ece413497b751f3b397)

@icewind1991 @schiesbn any idea ?

**Comment by PVince81** [comment]
Ok I know what is happening: there's a user "user2" who has never logged in, so that user has no actual home folder and the "files" folder hasn't been created yet.

Still, someone shared a folder with "user2" and modifies the files in it, which will propagate the etags for recipients. But since the recipient "user2" doesn't have a "files" folder, this message appears.

**Comment by PVince81** [comment]
Steps to reproduce:
1. Create two users "userx" and "usery"
2. Do NOT login as "usery"
3. Login as "userx"
4. Create a folder "test" and upload some files
5. Share "test" with "usery"
6. Upload more files into "test"
7. Check owncloud.log

Result:
```
{"reqId":"Pa2aBsnfCIBBhMO0BeDE","remoteAddr":"127.0.0.1","app":"files_sharing","message":"can not update etags on \/files\/ for user usery. Path does not exists","level":0,"time":"2015-04-10T14:47:35+00:00","method":"POST","url":"\/owncloud\/index.php\/core\/ajax\/share.php"}
{"reqId":"33J2LgyJi8BIwHalCs\/v","remoteAddr":"127.0.0.1","app":"files_sharing","message":"can not update etags on \/files\/ for user usery. Path does not exists","level":0,"time":"2015-04-10T14:47:40+00:00","method":"POST","url":"\/owncloud\/index.php\/apps\/files\/ajax\/upload.php"}
```

Then if you login once as "usery", the messages will not appear any more.

**Comment by rullzer** [comment]
This is fixed in 9.1 I think... @PVince81 could you verify?

**Comment by PVince81** [comment]
The message does not appear any more on master / 9.1
